docs/vexim-acl-check-content.conf

```diff
@@ -31,14 +31,14 @@
   # Reject mails with more than 15 spam-points. This is a system-wide setting and does not respect
   # individual user settings!
   #deny    message = This message has been rejected due to spam.
-  #        spam = maildeliver:true
+  #        spam = vmail:true
   #        condition = ${if >{$spam_score_int}{150}{true}{false}}
 
   # Always add X-Spam-Score and X-Spam-Report headers, using SA system-wide settings
   # (user "nobody"), no matter if over threshold or not.
```

Comment:
Interestingly, this line mentions 'nobody', this must be a mistake.
But we may want to use 'nobody' in non-packaged vexim versions.